l10n_it_delivery_note/models/stock_delivery_note.py

```diff
@@ -470,19 +470,6 @@ def _check_delivery_notes_before_invoicing(self):
                 raise UserError(
                     _("%s hasn't sale order!") % delivery_note_id.display_name
                 )
-            if (
-                len(
-                    delivery_note_id.mapped("sale_ids.picking_ids.picking_type_id.code")
-                )
-                > 1
-            ):
-                raise UserError(
-                    _(
-                        "Sale orders related to %s have return! "
-                        "For invoicing, go to sale orders."
-                    )
-                    % delivery_note_id.display_name
-                )
```

Comment:
Non è più necessario, blocca la fatturazione nel caso di ordini di vendita che hanno generato un IN per qualsiasi motivo.